app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,265 @@
+
+### Parallel Transition
+The [ParallelTransition](javafxdoc:ParallelTransition) plays a list of animations in parallel.
+
+![ParallelTransition](/assets/images/javafx/animation/transition/parallel-example.gif)
+```java
+Rectangle rectangle = new Rectangle(50, 50, 10, 10);
+rectangle.setFill(Color.GREEN);
+
+TranslateTransition translate = new TranslateTransition(Duration.seconds(5));
+translate.setToX(200);
+translate.setToY(200);
+translate.setInterpolator(Interpolator.LINEAR);
+
+RotateTransition rotate = new RotateTransition(Duration.seconds(5));
+rotate.setFromAngle(0);
+rotate.setToAngle(360);
+rotate.setInterpolator(Interpolator.LINEAR);
+
+ParallelTransition transition = new ParallelTransition(rectangle, translate, rotate);
+transition.play();
+```
+### Sequential Transition
+The [SequentialTransition](javafxdoc:SequentialTransition) plays a list of animations in sequential order. 
+It is not recommended to contain an `Animation`, which is not the last one, with `Duration.INDEFINITE` as this will block all later animations in the sequence.
+
+## Timeline
+A [Timeline](javafxdoc:Timeline) is used to define a free form `Animation` on any `WritableValue<T>`. It is helpful if none of the built-in transitions operate on the required properties.
+
+It consists of a sequential series of `KeyFrame`s.
+Each `KeyFrame` encapsulates a moment in time (**Cue Point**), and collectively specify how target properties evolve over the entire duration.
+
+> **Warning:** A running Timeline is being referenced from the FX runtime. Infinite Timeline might result in a memory leak if not stopped properly. All the objects with animated properties would not be garbage collected.
+
+### KeyFrame
+A [KeyFrame](javafxdoc:KeyFrame) represents a specific moment in an animation sequence (**Cue Point**) and comprises a collection of `KeyValue` instances that change from start to the given `Duration`.
+
+A KeyFrame can have a name which then can be used to identify this `KeyFrame` in an animation, for example for starting from this specific `KeyFrame` using `playFrom(String cuePoint)`.
+It is also possible to provide an `onFinished` implementation, which will be invoked when hitting this cue point.
+
+### KeyValue
+A [KeyValue](javafxdoc:KeyValue) establishes a mapping between a `WritableValue<T>` and a target value of type `T`. This is used to define the change of a value.
+An `Interpolator` can be additionally defined to set the rate of change for this value. The `KeyValue` class is immutable.
+
+### Example
+This example of `Timeline` creates a `Circle` which moves 200px in x direction over the duration of 5 seconds:
+
+![TimelineExample](/assets/images/javafx/animation/timeline-example.gif)
+```java
+Circle circle = new Circle(50, 150, 10, Color.GREEN);
+
+KeyValue x = new KeyValue(circle.translateXProperty(), 200);
+KeyFrame frame = new KeyFrame(Duration.seconds(5), x);
+Timeline timeline = new Timeline(frame);
+
+timeline.play();
+```
+
+## Interpolator
```

Comment:
This section could do with a code snippet showing how an interpolator is used. (Just one, no reason to have one per implementation.)